apps/style/common.scss

```diff
@@ -943,6 +943,15 @@ html[dir="RTL"] .prompt-icon-cell {
   .fa {
     font-size: 18px;
   }
+  /* We don't want some of these styles to affect modal content. */
+  .ml-modal {
+    span {
+      line-height: 12px;
+    }
+    .fa {
+      font-size: 12px;
+    }
+  }
```

Comment:
I wondered why this problem doesn't happen with the other settings cog dialogs, and it's because they all use different buttons 😭